index.html

```diff
@@ -0,0 +1,187 @@
+                    </p>
+                </div>
+            </article>
+            <article class="card">
+                <div class="card-image">
+                    <img src="./images/speaker.png" alt="">
+                </div>
+                <div class="card-title">
+                    <h3>Great speakers</h3>
+                </div>
+                <div class="card-desrc">
+                    <p>Passion will cause others to be conviced and enjoy the beliefs.
+                    </p>
+                </div>
+            </article>
+            <article class="card">
+                <div class="card-image">
+                    <img src="./images/swing.png" alt="">
+                </div>
+                <div class="card-title">
+                    <h3>have fun</h3>
+                </div>
+                <div class="card-desrc">
+                    <p>share good and enjoyable time with other tech women.
+                    </p>
+                </div>
+            </article>
+            <article class="card">
+                <div class="card-image">
+                    <img src="./images//pitch.png" alt="">
+                </div>
+                <div class="card-title">
+                    <h3>pitching</h3>
+                </div>
+                <div class="card-desrc">
+                    <p>present your project to sponsers and get fund.
+                    </p>
+                </div>
+            </article>
+            
+        </div>
+        <a class="join-us" href="">Join us</a>
+      
+        <a class="see-all" href="">SEE THE WHOLE PROGRAM
+            <hr>
+        </a>
+        
+       
+
+    </section>
+    <section class="feature">
+        <div class="feature-title">
+            <h2> Featured speakers</h2>
+            <hr>
+        </div>
+        <div class="feature-card-container">
+
+        </div>
+
+        <div class="see-more">
+            <a class="click" href=""> MORE</a>
+            <a href="">
+                <img src="./images/down.png" alt=""></a>
+
+        </div>
+    </section>
+    <section class="partners">
+        <div class="partners-title">
+        <h3>Partner</h3>
+        <hr>
+        </div>
+        <div class="partners-list" >
+            <ul>
+                <li><img src="./images/airbnbmain.png" alt=""></li>
+                <li><img src="./images/mircosoft.png" alt=""></li>
+                <li><img src="./images/google.png" alt=""></li>
+                <li><img id="amazon" src="./images/amazon.png" alt=""></li>
+            </ul>
+        </div>
+    </section>
+    <footer>
+        <div class="footers">
+            <a href=""><img src="./images/finalogo.png" alt=""></a>
+            <div class="copyright">
+               <p class="par">"Tech girls meetup (2023),"
+                
+                "All right Reserved."
+               </p>
+              
+            </div>
+        </div>
+        
+    </footer>
```

Comment:
- [ ] Kindly make sure this section doesn't appear on the mobile version as it is on Cindy's design.

![Screenshot from 2023-02-18 01-02-17](https://user-images.githubusercontent.com/97834160/219813615-942b0260-f21a-410b-8a47-b09e3bf67679.png)